mega-zep-backend/src/main/java/com/gepardec/mega/domain/calculation/time/HolidayCalculator.java

```diff
@@ -0,0 +1,45 @@
+package com.gepardec.mega.domain.calculation.time;
+
+import com.gepardec.mega.domain.calculation.AbstractTimeWarningCalculationStrategy;
+import com.gepardec.mega.domain.calculation.WarningCalculationStrategy;
+import com.gepardec.mega.domain.model.monthlyreport.ProjectEntry;
+import com.gepardec.mega.domain.model.monthlyreport.TimeWarning;
+import com.gepardec.mega.domain.model.monthlyreport.TimeWarningType;
+import de.jollyday.HolidayCalendar;
+import de.jollyday.HolidayManager;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HolidayCalculator extends AbstractTimeWarningCalculationStrategy implements WarningCalculationStrategy<TimeWarning> {
+    @Override
+    public List<TimeWarning> calculate(List<ProjectEntry> projectEntries) {
+        List<TimeWarning> warnings = new ArrayList<>();
+
+        if (projectEntries.isEmpty()) {
```

Comment:
isEmpty-Abfrage hier sinnlos. Wenn keine Einträge vorhanden sind, geht er übers forEach drüber und gibt das leere warnings-Array zurück.
Können die projectEntries nur leer oder auch null sein?

Reply:
Im  Normalbetrieb kann es glaube ich nur leer sein, soweit mir bekannt kommt null ins Spiel wenn man es absichtlich in einem Test setzt oder ähnliches. Hab  es jetzt entfernt.